### ISSUE TITLE
StringFragement moved into SequenceFragement

### ISSUE THREAD

**Issue by markushx** [martinthomson/abnf2regex#2]
The following rule

```
seq    = "[" ( "ab" / "cd" ) "]"
```

gets translated to 

```
seq = ("[" ("ab" / "cd" / "]"))
```

by abnf2regex. This breaks e.g. RFC3986 regexp generation, for e.g. IP-literal.

I have fixed this using the trivial patch:

```
diff --git a/abnf2regex/src/net/abnf2regex/SequenceFragment.java b/abnf2regex/src/net/abnf2regex/SequenceFragment.java
index 2692cb7..ee3590b 100755
--- a/abnf2regex/src/net/abnf2regex/SequenceFragment.java
+++ b/abnf2regex/src/net/abnf2regex/SequenceFragment.java
@@ -41,14 +41,14 @@ public class SequenceFragment extends GroupFragment
      */
     private boolean mergeWithLast(RuleFragment frag)
     {
-        if (this.fragments.size() > 0)
+        /*if (this.fragments.size() > 0)
         {
             RuleFragment last = this.fragments.getLast();
             if (last.getOccurences().equals(frag.getOccurences()))
             {
                 return last.append(frag);
             }
-        }
+        }*/
         return false;
     }
```

**Comment by martinthomson** [comment]
Fixed.  That was the right code to replace, and removing it was safe, but it's a little better to make it work.  That also fixed #1.  Let me know how this works out for you.

**Comment by markushx** [comment]
Commit 4576540 fixed it for me (in the proper way). Thanks!
